oap-server/pom.xml

```diff
@@ -373,6 +374,21 @@
                 <artifactId>nacos-client</artifactId>
                 <version>${nacos.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.curator</groupId>
+                <artifactId>curator-recipes</artifactId>
```

Comment:
You have put this in dependency management. I should adjust `pom.xml` in `cluster-zookeeper-plugin`, make version consistently in properties config.

Also, one question, what is the difference between `curator-recipes` and `curator-x-discovery`? Both of them come from `org.apache.curator`. The other one used in `cluster-zookeeper-plugin`.

Reply:
`curator-x-discovery` pom contains `curator-recipes`, I think `curator-x-discovery` implement a naming service API by `curator-recipes`.

remove curator-recipes and use cluster-zookeeper-plugin is more better

Reply:
Then, move the `curator-x-discovery` to server root pom.xml dependency management.

Reply:
I also move `curator-test` at `cluster-zookeeper-plugin` to `oap-server ` pom.xml